bedrock/firefox/templates/firefox/election/index.html

```diff
@@ -16,152 +16,298 @@
 {% block body_class %}election{% endblock %}
 
 {% block content %}
-
 <main class="mzp-t-firefox">
-  <div id="case-not-firefox-desktop" class="hidden">
-    <div class="t-paper">
-      <section class="mzp-c-hero">
+  {% if switch('firefox-election-build') %}
+    <div id="case-not-firefox-desktop" class="hidden">
+      <div class="t-paper">
+        <section class="mzp-c-hero">
+          <div class="mzp-l-content">
+            <div class="mzp-c-hero-body">
+              <h1 class="mzp-c-hero-title">Get Smart Before You Vote.</h1>
+              <div class="mzp-c-hero-desc">
+                <p>Download the Firefox Election Bundle to get the already powerful Firefox browser supercharged with extensions that help you decode political ad targeting and keep Facebook off your trail.</p>
+              </div>
+            </div>
+          </div>
+        </section>
+        <section class="mzp-c-call-out mzp-t-product-firefox mzp-t-firefox t-container">
+          <div class="mzp-l-content">
+            <h2 class="mzp-c-call-out-title">The Firefox Facebook Container</h2>
+            <div class="mzp-c-call-out-desc">
+              <p>Facebook lets political advertisers target you by tracking your activity on and off their platform.</p>
+              <p>With this extension, Firefox makes it easier to explore the web without Facebook watching, so you can keep your political opinions to yourself and your dinner companions.</p>
+              <p class="mzp-c-button-download">
+                <small class="mzp-c-button-download-privacy-link">
+                  <a href="{{ url('privacy.notices.firefox') }}">Firefox Privacy Notice</a>
```

Comment:
Do we need the privacy link here? It seems out of place without a button.

Reply:
It's in the requirements 🤷‍♀️